actions/form/class.UserSettings.php

```diff
@@ -142,6 +145,22 @@ protected function initElements()
         }
 
         $this->addTimezoneEl($this->form);
+
+        if (
+            $this->getFeatureFlagChecker()->isEnabled(
+                FeatureFlagCheckerInterface::FEATURE_FLAG_SOLAR_DESIGN_ENABLED
+            )
+        ) {
+            $this->addInterfaceModeEl($this->form);
+        }
+    }
+
+    private function addInterfaceModeEl($form): void
+    {
+        $interfaceModeElement = tao_helpers_form_FormFactory::getElement('interface_mode', 'Radiobox');
```

Comment:
suggestion: Lets use constants for better tracking

```suggestion
        $interfaceModeElement = tao_helpers_form_FormFactory::getElement(UserSettingsInterface::INTERFACE_MODE, 'Radiobox');
```